R/utils.R

```diff
@@ -99,12 +99,16 @@ expose_stan_fns <- function(files, target_dir, ...) {
 #' functionality without needing to install `cmdstanr`.
 #'
 #' @param type A character string indicating the example to load.
-#' Supported options are "nowcast, "preprocessed_observations", "observations",
-#' and "script" which are the output of epinowcast()], [enw_preprocess_data()],
-#' and [enw_latest_data()] applied to the [germany_covid19_hosp] package
-#' dataset), and the script used to generate these examples respectively.
+#' Supported options are
```

Comment:
Is this format as intended? seems odd to have a bullet point list here?

Reply:
intended; i've proposed changes to make it more bullet-y.